test/add.spec.js

```diff
@@ -0,0 +1,30 @@
+const Calculator = require('../unit_testing_framework/calculator/calculator');
+const {expect} = require('chai');
+
+describe('Tests for method add()', function() {
+    let calculator;
+
+    beforeEach(function() {
```

Comment:
after each with `null` for calculator variable to be sure that every test is independent

Reply:
Added afterEach() into add.spec.js and multiply.spec.js